crates/storage/trie/node_hash.rs

```diff
@@ -1,4 +1,4 @@
-use std::{cmp::min, default};
+use std::{cmp::min, default, mem::size_of};
```

Comment:
remove?

Reply:
Code didn't compile without that modification.

Reply:
locally for you or in the ci?

Reply:
Do you have this rust version? https://github.com/lambdaclass/ethereum_rust/blob/main/.tool-versions

Reply:
I had an older version. Seems this no longer happens on `1.80.1`.